packages/rum/src/rum.ts

```diff
@@ -205,11 +199,17 @@ export function startRum(
         lifeCycle.notify(LifeCycleEventType.CUSTOM_ACTION_COLLECTED, { context, name, type: UserActionType.CUSTOM })
       }),
       getInternalContext: monitor((startTime?: number): InternalContext | undefined => {
-        const view = parentContexts.findView(startTime)
-        if (session.isTracked() && view && view.sessionId) {
+        const viewContext = parentContexts.findView(startTime)
+        if (session.isTracked() && viewContext && viewContext.sessionId) {
           return (withSnakeCaseKeys(deepMerge(
-            { applicationId },
-            view,
+            {
+              applicationId,
+              sessionId: viewContext.sessionId,
+              view: {
+                id: viewContext.view.id,
+                url: viewContext.view.url,
```

Comment:
what about passing referrer in internal context too?

Reply:
I'm not sure this is valuable, but I don't mind adding it, it would save us a few lines of code

Reply:
it could be nice to have the same info in related trace/log